packages/server/graphql/types/User.ts

```diff
@@ -497,10 +511,7 @@ const User: GraphQLObjectType<any, GQLContext> = new GraphQLObjectType<any, GQLC
             }
           }
         }
-        return r
-          .table('RetroReflectionGroup')
-          .getAll(r.args(Array.from(currentResultGroupIds)), {index: 'id'})
-          .run()
+        return dataLoader.get('retroReflectionGroups').loadMany(Array.from(currentResultGroupIds))
```

Comment:
loadMany could return an error. i made a helper function to aid with this.  see https://github.com/ParabolInc/parabol/blob/9c27f5716901081c321d69ea6f56fb4756a83c1d/packages/server/graphql/isValid.ts